test/Microsoft.Dnx.ApplicationHost.FunctionalTests/AppHostTests.Old.cs

```diff
@@ -0,0 +1,309 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Versioning;
+using Microsoft.Dnx.CommonTestUtils;
+using Microsoft.Dnx.Tooling;
+using Microsoft.Dnx.Runtime;
+using Microsoft.Dnx.Runtime.Common.Impl;
+using NuGet;
+using Xunit;
+
+namespace Microsoft.Dnx.ApplicationHost.Old
+{
+    [Collection("ApplicationHostTestCollection")]
+    public class AppHostTests
+    {
+        private readonly DnxRuntimeFixture _fixture;
+
+        public AppHostTests(DnxRuntimeFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        public static IEnumerable<object[]> RuntimeComponents
+        {
+            get
+            {
+                return TestUtils.GetRuntimeComponentsCombinations();
+            }
+        }
+
+        [Theory]
+        [MemberData(nameof(RuntimeComponents))]
+        public void AppHostReturnsNonZeroExitCodeWhenNoSubCommandWasGiven(string flavor, string os, string architecture)
+        {
+            var runtimeHomeDir = _fixture.GetRuntimeHomeDir(flavor, os, architecture);
+
+            string stdOut, stdErr;
+            var exitCode = TestUtils.ExecBootstrapper(
+                runtimeHomeDir,
+                arguments: string.Empty,
+                stdOut: out stdOut,
+                stdErr: out stdErr);
+
+            Assert.NotEqual(0, exitCode);
```

Comment:
I will need to make a new commit to port over the changes from `AppHostTests.cs` to `AppHostTests.Old.cs`